addOns/automation/src/main/java/org/zaproxy/addon/automation/AutomationStatsListener.java

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
I'm not sure this class is needed. We can reply in the ZAP in-memory stats as they record exactly the same things. If they have been disabled then we can just warn that the tests wont work.